modules/util/client-extension-util-spring-boot/build.gradle

```diff
@@ -1,5 +1,5 @@
 dependencies {
 	api group: "org.springframework.boot", name: "spring-boot-starter-oauth2-client", version: "2.7.9"
-	api group: "org.springframework.boot", name: "spring-boot-starter-oauth2-resource-server", version: "2.7.9"
+	api group: "org.springframework.boot", name: "spring-boot-starter-oauth2-resource-server", version: "2.7.11"
```

Comment:
We should bump the rest of the spring boot references to 2.7.11 as well to make sure everything is still compatible.

Reply:
@gamerson I have sent a new pull that updates all spring-boot's libraries from 2.7.9 to 2.7.11
https://github.com/liferay-frontend/liferay-portal/pull/3616

Reply:
I will leave https://github.com/liferay-frontend/liferay-portal/pull/3616 open for now, but for future PRs like this feel free to send to @gamerson directly as his team is the maintainers of these portion of client extensions.

Reply:
Hi @bryceosterhaus I have sent the pull request to @gamerson 
https://github.com/gamerson/liferay-portal/pull/622